### PR TITLE
Added steps for configuring an Okta tile 

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/264

Updated Ralph's original wording suggestion slightly based on my understanding of what an Okta tile is. If that's not right, let me know @rbankston 

Once this is approved, I'll push it to other doc versions.

### CODE REVIEW

**Review by rbankston — COMMENTED** [review]
@jwitz The tile can appear if they set that or not. What if we just change that wording to (_Optional_) Set your Initiate login URI to `https://houston.BASEDOMAIN/v1/oauth/start?provider=okta`. The tile can appear without that set correctly but it wouldn't work without that there.

**Comment by jwitz** [comment]
Sounds good @rbankston, I'm assuming that someone with access to this in Okta would know whether or not to configure the URI themselves - we'll leave that to the Okta docs and focus on the actual configuration here. Made changes based on your comment, so this should be good to go now!

And just to confirm (and because I'm paranoid): There's no ending `/` on the URI like there is for the redirect URI, correct? That was an issue last week.

**Comment by rbankston** [comment]
Screen Shot 2021-02-22 at 2 44 42 PM

Is a screenshot of my working login section

**Review by paolaperaza — APPROVED** [review]
Got a little behind on this review, but this looks good @jwitz. Feel free to merge.

File: enterprise/next/06_manage-astronomer/01_integrate-auth-system.md [hunk 1]
```diff
@@ -108,15 +108,17 @@ Follow the steps below.
...
+1. If you haven't already, create an [Okta account](https://www.okta.com/).
 
-2. Create a new web app in your Okta account for Astronomer
+2. In your Okta account, create a new web app for Astronomer.
 
-3. Set your `login redirect URI` to be `https://houston.BASEDOMAIN/v1/oauth/redirect/`, where the `BASEDOMAIN` is the domain at which you're hosting your Astronomer installation
+3. Set your `login redirect URI` to be `https://houston.BASEDOMAIN/v1/oauth/redirect/`, where the `BASEDOMAIN` is the domain at which you're hosting your Astronomer installation.
 
-4. Enable `Implicit (Hybrid)` Flow on the Okta application
+4. Enable `Implicit (Hybrid)` Flow on the Okta application.
```
Comment (paolaperaza) [inline enterprise/next/06_manage-astronomer/01_integrate-auth-system.md:117]:
```suggestion
4. Under **Allowed grant types**, enable `Implicit (Hybrid)`.
```

File: enterprise/next/06_manage-astronomer/01_integrate-auth-system.md [hunk 2]
```diff
@@ -108,15 +108,17 @@ Follow the steps below.
...
 
-4. Save the `Client ID` generated for this Okta app for use in the next steps
+5. Save the `Client ID` generated for this Okta app for use in the next steps.
+
+6. Set your Initiate Login URI to `https://houston.BASEDOMAIN/v1/oauth/start?provider=okta`  (_Optional_).
```
Comment (paolaperaza) [inline enterprise/next/06_manage-astronomer/01_integrate-auth-system.md:121]:
```suggestion
6. To ensure that an Okta tile appears, set `Initiate Login URI` to `https://houston.BASEDOMAIN/v1/oauth/start?provider=okta`  (_Optional_).
```

**Review by paolaperaza — COMMENTED** [review]
Actually, took a look at the screenshots Ralph sent over and made some copy suggestions. Still approving, but feel free to incorporate them!

**Pull request merged**